server/app.ts

```diff
@@ -16,6 +20,16 @@ app.use([
 ]);
 
 app.use('/api/v1', router);
+
+app.use(express.static(join(__dirname, '..', 'client', 'dist')));
```

Comment:
wrap this part in` if statement` to execute the code depends on the `nodeEnv` status